tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,53 @@
+- name: Test agent configuration
+  hosts: wazuh-windows
+  tasks:
+
+    - name: Create temp folder
+      win_file:
+        path: C:\temp
+        state: directory
+
+    - name: Copy ossec.conf
+      ansible.windows.win_copy:
+        src: C:\Program Files (x86)\ossec-agent\ossec.conf
+        dest: C:\temp
+        remote_src: true
+
+    - name: Enable the agent module to collect installed packages (Windows)
```

Comment:
Name is not correct

Reply:
Done in f1c9938c96b60a5587ab444a4da2ecd55451d839